cli/src/cli_util.rs

```diff
@@ -2707,6 +2708,14 @@ fn resolve_aliases(
             real_commands.insert(alias.to_string());
         }
     }
+    for alias in aliases_map.keys() {
+        if real_commands.contains(alias) {
+            return Err(user_error(format!(
+                "Cannot define alias for built-in command '{alias}'"
```

Comment:
nit: "Cannot redefine" or "Cannot override" to clarify the alias name matters?

Reply:
Perhaps "Cannot define an alias that overrides the built-in command '{alias}'"?